src/service/resources-impl.js

```diff
@@ -1008,7 +1008,7 @@ export class Resources {
       // Note that when the document is not visible, neither are any of its
       // elements to reduce CPU cycles.
       const shouldBeInViewport = (this.visible_ && r.isDisplayed() &&
-          r.overlaps(visibleRect));
+          (r.isFixed() || r.overlaps(visibleRect)));
```

Comment:
I'd prefer if the `isFixed` call would be in `overlaps` and `overlaps` would use a `0,0` rect in that case. That way stuff that is fixed but at `-1000px` does not get rendered.